packages/go/cypher/analyzer/analyzer.go

```diff
@@ -125,25 +152,31 @@ func (s *ComplexityMeasure) onProjection(node *model.Projection) {
 		// Distinct incurs a weight since it will change how the projection is materialized
 		s.Weight += Weight1
 	}
+
+	return nil
 }
 
-func (s *ComplexityMeasure) onPartialComparison(node *model.PartialComparison) {
+func (s *ComplexityMeasure) onPartialComparison(stack *model.WalkStack, node *model.PartialComparison) error {
```

Comment:
I can't remember if it's a compiler warning or not but you may want to consider renaming "stack" to "_" since that param is unused.

Reply:
Patched up